beacon-chain/powchain/engine_client.go

```diff
@@ -287,6 +288,13 @@ func handleRPCError(err error) error {
 	}
 	e, ok := err.(rpc.Error)
 	if !ok {
+		if strings.Contains(e.Error(), "Unauthorized") {
```

Comment:
Do we think we can specifically check errors better here like use `error.Is`?  Parsing `Unauthorized` in the error log has a lot of invariants

Reply:
It's a hard one because it isnt a specific RPC error as defined in the rpc spec. 401 Unauthorized seems a lot more narrow, what do you think?